src/directive.js

```diff
@@ -2,7 +2,7 @@
  * Based on gg.editableText, originally created by Gabriel Grinberg on 6/13/14.
  */
 
-(function() {
+(function () {
```

Comment:
Not on purpose, no. There's some cleanup that should be done at some point to this project, but I figure there's little point in doing the JSCS cleanup since it's a deprecated project. I'll add a ticket for the backlog.